src/component/page-list/page-tile-composite.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import { observer } from 'mobx-react';
+import { PreviewTile, PreviewTileProps } from '../../lsg/patterns/preview-tile';
+import Space, { Size } from '../../lsg/patterns/space';
+
+export const PageTileComposite: React.StatelessComponent<PreviewTileProps> = observer(
```

Comment:
Does it make sense to have a composite component when its only purpose is a spacer? Much boiler-plate code here. Maybe put that composition code into the PreviewTile or container?

Reply:
You are right. The `page-tile-composite.tsx ` is gone :)